fapolicy_analyzer/ui/actions.py

```diff
@@ -168,3 +192,80 @@ def init_system() -> Action:
 
 def system_initialized() -> Action:
     return _create_action(SYSTEM_INITIALIZED)
+
+
+def init_daemon() -> Action:
+    return _create_action(INIT_DAEMON)
+
+
+def daemon_initialized() -> Action:
+    return _create_action(DAEMON_INITIALIZED)
+
+
+def request_daemon_reload() -> Action:
+    return _create_action(REQUEST_DAEMON_RELOAD)
+
+
+def received_daemon_reload() -> Action:
+    return _create_action(RECEIVED_DAEMON_RELOAD)
+
+
+def error_daemon_reload(error: str) -> Action:
+    return _create_action(ERROR_DAEMON_RELOAD, error)
+
+
+def request_daemon_status() -> Action:
+    logging.debug("request_daemon_status() -> Action: REQUEST_DAEMON_STATUS")
+    return _create_action(REQUEST_DAEMON_STATUS)
+
+
+def received_daemon_status(status: bool) -> Action:
+    logging.debug(f"received_daemon_status({status}) -> Action: RECEIVED_DAEMON_STATUS")
+    return _create_action(RECEIVED_DAEMON_STATUS, status)
+
+
+def error_daemon_status(error: str) -> Action:
+    return _create_action(ERROR_DAEMON_STATUS, error)
+
+
+def request_daemon_status_update():
+    logging.debug("request_daemon_status_update()")
+    return _create_action(REQUEST_DAEMON_STATUS_UPDATE)
+
+
+def received_daemon_status_update(bStatus: bool):
```

Comment:
As far as I can tell this guy is never used.

Reply:
It was used but did not play well with the manual Start/Stop iirc.It will be removed.

Reply:
Removed code associated with REQUEST_DAEMON_STATUS_UPDATE action.